### PR TITLE
Update web readme

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: web/readme.md [hunk 1]
```diff
@@ -1,14 +1,22 @@
 Samples for [Flutter for web](https://flutter.dev/web).
 
-#### Building Samples code
-Set up `webdev` for first time as below:
-```
-$ pub global activate webdev
-$ pub get
-```
+## See built samples
```
Comment (RedBrogdon) [inline web/readme.md]: Consider "See the samples in action"

File: web/readme.md [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
 
-Build and run code locally at `http://localhost:8080`
-```
+All of the samples are at [flutter.github.io/samples](https://flutter.github.io/samples/).
```
Comment (RedBrogdon) [inline web/readme.md]: Consider "Compiled versions of the samples are hosted at...."

File: web/readme.md (hunk 1 above)
Comment (kevmoo) [reply web/readme.md]: 👍

File: web/readme.md (hunk 2 above)
Comment (kevmoo) [reply web/readme.md]: 👍

**Pull request merged**